src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -220,6 +222,7 @@ public sealed partial class ComEventInterfaceAttribute : System.Attribute
         public System.Type EventProvider { get { throw null; } }
         public System.Type SourceInterface { get { throw null; } }
     }
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows7.0")]
```

Comment:
Is it intentional that some of the attributes use `windows7.0` and some just `windows` ?

Reply:
@jkotas We should not use OS version for marking stuff as Windows-specific, this is why I've found all such usages and fixed them in https://github.com/dotnet/runtime/pull/40610/commits/5b1e62c7a5f87976f045455804e11e6e92760f24